app/models/dream.rb

```diff
@@ -33,51 +38,36 @@ def positivity
     end
   end
 
-  def get_sentiment
+  def dream_analysis
     alchemyapi = AlchemyAPI.new
-    response = alchemyapi.sentiment('text', self.text, { 'sentiment'=>1 })
-
-    if response['status'] == 'OK'
-      sentiment = response['docSentiment']['score']
+    keyword_response = alchemyapi.keywords('text', text, 'sentiment' => 1)
+    sentiment_response = alchemyapi.sentiment('text', text, { 'sentiment'=>1 })
+    if sentiment_response['status'] == 'OK' && keyword_response['status'] == 'OK'
+      update_sentiment(sentiment_response)
+      update_keywords(keyword_response)
     else
-      puts 'Error in sentiment extraction call: ' + response['statusInfo']
+      puts 'Error in sentiment extraction call: ' + keyword_response['statusInfo']
```

Comment:
Line is too long. [82/80]<br>Do not write to stdout. Use Rails' logger if you want to log.